tests/validation/tests/v3_api/test_cert_rotation.py

```diff
@@ -1,18 +1,28 @@
 import pytest
 import datetime
 import time
-from .common import get_user_client_and_cluster
-from .common import validate_cluster_state
-from .common import get_etcd_nodes
-from .common import wait_for_nodes_to_become_active
-from .common import wait_for_node_status
+import os
+from .common import *  # NOQA
+from rancher import ApiError
 
 # Globals
 # Master list of all certs
 ALL_CERTS = ["kube-apiserver", "kube-controller-manager",
              "kube-node", "kube-proxy", "kube-scheduler",
              "kube-etcd", "kube-ca"]
 
+# --------------------- rbac test -----------------------
+@if_test_rbac
+@pytest.mark.parametrize("role", [CLUSTER_MEMBER, PROJECT_OWNER, PROJECT_MEMBER, PROJECT_READ_ONLY])
```

Comment:
Can we also include "CLUSTER_OWNER" ?

Reply:
The default tests are already being run by cluster owner. But we can include it in the rbac tests.

Reply:
Check for cluster_owner implemented.